packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -552,6 +583,17 @@ export type TimeZoneUtc = 'utc';
  */
 export type TimeZoneBrowser = 'browser';
 
+export interface DataSourceRef {
```

Comment:
This is already defined in the dashboard types so we need to remove one of them.

Reply:
ahh, our first problem with the shared namespacing!

Reply:
Actually it seems to be the same type just defined in two places, had the same issue for another panel.

Reply:
Seems like the Levitate check fails due to DataSourceRef being exported twice.

Reply:
And the frontend lint step